rpm/fedora/rust-keylime-allow-setting-mb-log-path.patch

```diff
@@ -0,0 +1,41 @@
+diff --git a/keylime-agent/src/main.rs b/keylime-agent/src/main.rs
```

Comment:
It is the same patch AFAICS, maybe you can make it a link, of put them in `/rpm/patch/` and make both links to it.

Reply:
OK, I can do this.

Reply:
In the end, I think it is safe to make the code part of the project instead of bringing as a separate patch.

Reply:
Yeah ... that is a sensible option